jina/clients/mixin.py

```diff
@@ -61,14 +61,17 @@ async def _get_results(*args, **kwargs):
                 return result
 
         with ExitStack() as stack:
-            if self.args.infrastructure == InfrastructureType.K8S:
+            if (
+                not hasattr(self.args, 'infrastructure')
```

Comment:
why it should be the case??? It always has them no?

Reply:
not when running a client directly